go/flags/endtoend/vtgate.txt

```diff
@@ -54,7 +54,7 @@ Flags:
       --discovery_low_replication_lag duration                           Threshold below which replication lag is considered low enough to be healthy. (default 30s)
       --emit_stats                                                       If set, emit stats to push-based monitoring and stats backends
       --enable-partial-keyspace-migration                                (Experimental) Follow shard routing rules: enable only while migrating a keyspace shard by shard. See documentation on Partial MoveTables for more. (default false)
-      --enable-udfs                                                      Enable UDFs support in vtgate.
```

Comment:
Does this need release notes/change log entry?

Reply:
Good catch!